frontend/openchat-worker/src/worker.ts

```diff
@@ -62,18 +62,18 @@ async function initialize(
         return { kind: "auth_identity_not_found" };
     }
 
-    authPrincipalString = authPrincipal.toString();
-    const ocIdentity = await ocIdentityStorage.get(authPrincipalString);
-    if (ocIdentity !== undefined) {
-        return { kind: "success", identity: ocIdentity };
-    }
-
     identityAgent = new IdentityAgent(
         authProviderIdentity as SignIdentity,
         identityCanister,
         icUrl,
     );
 
+    authPrincipalString = authPrincipal.toString();
```

Comment:
Why was this moved later? It looks unchanged so it may as well be earlier so that we can exit earlier if we have the OC identity.

Reply:
Because I need the identityAgent to exist so that I can look up a user's auth principals.